ext/Errno/t/Errno.t

```diff
@@ -14,29 +14,33 @@ BAIL_OUT("No errno's are exported") unless @Errno::EXPORT_OK;
 my $err = $Errno::EXPORT_OK[0];
 my $num = &{"Errno::$err"};
 
-is($num, &{"Errno::$err"});
+is($num, &{"Errno::$err"},
+    'element in @Errno::EXPORT_OK found via sub call');
 
 $! = $num;
-ok(exists $!{$err});
+ok(exists $!{$err}, 'error found in %!');
 
 $! = 0;
-ok(! $!{$err});
+ok(! $!{$err}, 'no true value for error in %!');
 
```

Comment:
These describe what the ok() is doing, but don't really describe the point of the test - in each case whether the entry in `%!` reflects the current value of `$!`.

Reply:
Adapted test descriptions in 80f77bb58d; please re-review.

Reply:
Good enough.